src/network/networkctl.c

```diff
@@ -684,7 +703,7 @@ static int list_links(int argc, char *argv[], void *userdata) {
 
         (void) pager_open(arg_pager_flags);
 
-        table = table_new("idx", "link", "type", "operational", "setup");
+        table = table_new("idx", "link", "type", "operational", "online", "setup");
```

Comment:
Not sure, but maybe "online" state at the end maybe ok for networkd-dispatcher.service??

Reply:
I did this, but it didn't seem to help... I'll leave it at the end of the line now though, since it doesn't really matter.